utils/k6_helpers.py

```diff
@@ -28,10 +25,30 @@ def k6_prepare_accounts(network, network_object, users, balance, bank_account):
     with open('./loadtesting/k6/data/accounts.json', 'w', encoding='utf-8') as f:
         json.dump(accounts, f)
 
+def deploy_erc20_contract(web3_client, faucet, account):
+    erc20 = ERC20(
```

Comment:
wrong ERC20 ...
It is better to check erc20ForSpl

Reply:
we will add erc20ForSpl scenario later